Python/ceval_gil.c

```diff
@@ -95,11 +96,11 @@ RESET_GIL_DROP_REQUEST(PyInterpreterState *interp)
 
 
 static inline void
-SIGNAL_PENDING_CALLS(PyInterpreterState *interp)
+SIGNAL_PENDING_CALLS(struct _pending_calls *pending, PyInterpreterState *interp)
```

Comment:
Want to make this and the unsignal version less SHOUTY, as it isn't a macro anymore?

Reply:
Also, the name "SIGNAL" is a bit confusing as this doesn't handle signals, but code around here does.

Reply:
The same applies to all the other static inline functions that used to be macros.  I'd rather leave renaming them all to a separate change.  I'm not sure the churn is worth it either.